superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -293,7 +293,10 @@ function DashboardList(props: DashboardListProps) {
                       className="action-button"
                       onClick={confirmDelete}
                     >
-                      <Icon name="trash" />
+                      <Icon
+                        {...{ 'data-test': 'dashboard-list-trash-icon' }}
```

Comment:
Wondering again about the need spread syntax over the attribute, which could be stripped out by babel for sure.